app/controllers/jobs_controller.rb

```diff
@@ -0,0 +1,18 @@
+class JobsController < ApplicationController
+  before_action :set_job, only: [:show, :edit, :update, :destroy]
```

Comment:
I can't see edit, update & destroy here. Perhaps might even move the before_action back to #show